tofhir-server/src/main/scala/io/tofhir/server/service/fhir/FileSystemTreeStructureService.scala

```diff
@@ -21,13 +21,19 @@ class FileSystemTreeStructureService {
 
   /**
    * Get the folder tree structure of the given base path
-   * @param basePath The base path to start with
+   * @param basePathString The base path to start with
    * @param includeFiles if false, only folders will be included in the tree
    * @return
    */
-  def getFolderTreeStructure(basePath: String, includeFiles: Boolean): FilePathNode = {
+  def getFolderTreeStructure(basePathString: String, includeFiles: Boolean): FilePathNode = {
     var fileCount = 0 // Track total number of files processed
 
+    val path = FileUtils.getPath(basePathString)
+    val basePath = FileUtils.getPath(basePathString)
```

Comment:
basePath and path variables are the same. Shall we remove this one ?

Reply:
path variable is removed.